configs/rpt/usbradio.conf

```diff
@@ -176,6 +176,11 @@ clipledgpio = 0             ; Enable ADC Clip Detect feature to use a GPIO outpu
                             ; Supports URI Clip LED by setting a GPIO (if available) high for 500mS
                             ; when clipping detected. Value = GPIO# to use (GPIO1 recommended)
 
+legacyaudioscaling = no     ; If yes, continue to do raw audio sample scaling and clipping, resulting in Tx audio levels increasing
```

Comment:
Add here that the default is currently yes but will eventually be no and/or setting removed